www/perl-lib/SMT/NCCRegTools.pm

```diff
@@ -103,8 +103,30 @@ sub new
     $self->{URI}      = $ruri;
     $self->{AUTHUSER} = $user;
     $self->{AUTHPASS} = $pass;
-    bless($self);
 
+    my $regsharing = SMT::Utils::hasRegSharing();
```

Comment:
I do not like that "hasRegSharing" returns a string true/false which both is true if you check the value in the if statement.
If you read the code you have the feeling that you always get an error if regsharing is disabled.
See also comment on Utils.pm